lib/bike_brigade_web/live/itinerary_live/index.ex

```diff
@@ -0,0 +1,72 @@
+defmodule BikeBrigadeWeb.ItineraryLive.Index do
+  use BikeBrigadeWeb, :live_view
+
+  alias BikeBrigade.LocalizedDateTime
+  alias BikeBrigade.Riders
+
+  import BikeBrigadeWeb.CampaignHelpers
+
+  import BikeBrigade.Utils, only: [humanized_task_count: 1]
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(:page, :itinerary)
+     |> assign(:page_title, "Itinerary")}
+  end
+
+  @impl true
+  def handle_params(params, _url, socket) do
+    {:noreply, apply_action(socket, socket.assigns.live_action, params)}
+  end
+
+  defp apply_action(socket, :index, params) do
+    date =
+      case params do
+        %{"date" => date} -> Date.from_iso8601!(date)
+        _ -> LocalizedDateTime.today()
+      end
+
+    load_itinerary(socket, date)
+  end
+
+  defp load_itinerary(socket, date) do
+    rider_id = socket.assigns.current_user.rider_id
+
+    if rider_id do
+      socket
+      |> assign(
+        :campaign_riders,
+        Riders.get_itinerary(rider_id, date)
+      )
+    else
+      socket
+      |> assign(:campaigns, [])
```

Comment:
```suggestion
      |> assign(:campaign_riders, [])
```

I missed updating this, which actually makes one of our tests fail, so good to have test :)